Source/Core/VideoBackends/Vulkan/TextureConverter.cpp

```diff
@@ -811,36 +782,37 @@ bool TextureConverter::CompileYUYVConversionShaders()
   static const char YUYV_TO_RGB_SHADER_SOURCE[] = R"(
     layout(std140, push_constant) uniform PCBlock
     {
-      int buffer_offset;
-      int src_stride;
+      uint buffer_offset;
+      uint src_stride;
     } PC;
 
+    IMAGE_BINDING(rgba8, 0) uniform writeonly image2DArray output_image;
     TEXEL_BUFFER_BINDING(0) uniform samplerBuffer source;
-    layout(location = 0) in vec3 uv0;
-    layout(location = 0) out vec4 ocol0;
+
+    layout(local_size_x = 8, local_size_y = 8) in;
 
     void main()
     {
-      ivec2 uv = ivec2(gl_FragCoord.xy);
-      int buffer_pos = PC.buffer_offset + uv.y * PC.src_stride + (uv.x / 2);
-      vec4 c0 = texelFetch(source, buffer_pos);
+      uvec2 uv = gl_GlobalInvocationID.xy;
+      int buffer_pos = int(PC.buffer_offset + (uv.y * PC.src_stride) + (uv.x / 2));
+      vec4 yuyv = texelFetch(source, buffer_pos);
 
-      float y = mix(c0.r, c0.b, (uv.x & 1) == 1);
+      float y = mix(yuyv.r, yuyv.b, (uv.x & 1u) == 1u);
```

Comment:
mix() isn't cheap and I doubt compilers optimize this properly for booleans, just do a ternary if